src/common/components/discussion/_index.scss

```diff
@@ -312,6 +320,24 @@
   }
 }
 
+.popper-discussion {
+  width: 75% !important;
+  z-index: 2 !important;
+  padding: 30px 0px;
+  @media (max-width: $sm-break) {
+    width: 100% !important;
+    background-color: rgba(0, 0, 0, 0.1);
```

Comment:
Is this color not in theme? And we need the same in both themes? Also, if we need 2 variants in both themes differently, can we add it in theme colors?